CH06_AudioConverter/CH06_AudioConverter/main.c

```diff
@@ -6,7 +6,7 @@
 #endif
 
 
-#define kInputFileLocation	CFSTR("/Insert/Path/To/Audio/File.xxx")
+#define kInputFileLocation	CFSTR("/Users/alexanderbollbach/Desktop/qt.m4a")
```

Comment:
Should revert this back to the generic path '/Insert/Path/To/Audio/File.xxx', since other users won't have an 'alexanderbollbach' directory, and it's better for them to blow up on a self-explanatory problem if they forget to reset the 'kInputFileLocation'.